configs/config.msi_ms7d25_ddr4

```diff
@@ -7,25 +7,24 @@ CONFIG_IFD_BIN_PATH="3rdparty/dasharo-blobs/$(MAINBOARDDIR)/descriptor.bin"
 CONFIG_ME_BIN_PATH="3rdparty/dasharo-blobs/$(MAINBOARDDIR)/me.bin"
 CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x100000
 CONFIG_EDK2_BOOT_TIMEOUT=3
+CONFIG_PCIEXP_DEFAULT_MAX_RESIZABLE_BAR_BITS=37
 CONFIG_HAVE_IFD_BIN=y
 CONFIG_BOARD_MSI_Z690_A_PRO_WIFI_DDR4=y
 # CONFIG_PCIEXP_ASPM is not set
 # CONFIG_PCIEXP_L1_SUB_STATE is not set
 # CONFIG_PCIEXP_CLK_PM is not set
 CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
+CONFIG_TPM_MEASURED_BOOT=y
 CONFIG_POWER_STATE_OFF_AFTER_FAILURE=y
 CONFIG_INCLUDE_HSPHY_IN_FMAP=y
 CONFIG_SOC_INTEL_COMMON_OC_WDT_ENABLE=y
 CONFIG_ENABLE_EARLY_DMA_PROTECTION=y
 CONFIG_HAVE_ME_BIN=y
 CONFIG_DASHARO_PREFER_S3_SLEEP=y
 CONFIG_PCIEXP_SUPPORT_RESIZABLE_BARS=y
-CONFIG_PCIEXP_DEFAULT_MAX_RESIZABLE_BAR_BITS=37
 CONFIG_PCIEXP_LANE_ERR_STAT_CLEAR=y
 CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
-CONFIG_DRIVERS_EFI_UPDATE_CAPSULES=y
```

Comment:
This suggests that capsules are enabled by default, I thought that no default meant disabled. I think this should be opt-in, at least for now.

Reply:
Good thing you've spotted it.  I just did it backwards: added option to defconfigs, re-generated them, then updated Kconfig to add the option.  No wonder it got removed.  Re-ordered commits and resaved configs.